src/library_fs.js

```diff
@@ -936,9 +936,6 @@ FS.staticInit();
         timestamp: Date.now()
       });
     },
-    lchmod(path, mode) {
-      FS.chmod(path, mode, true);
-    },
```

Comment:
Why remove this? It is part of the public API I think.

Reply:
Its not needed internally anywhere.  i.e. none of the syscalls depend on this since (they instead depend on the third argument to `FS.chmod`).

You would argue that its part of the public API, but to be fair this syscall is so unusual its very unlikely anyone is using it.  For example, its not supported by linux or by node.

Reply:
I reverted this change for now.